content/docs/other-guides/authentication.md

```diff
@@ -0,0 +1,514 @@
++++
+title = "Kubeflow Authentication with Istio and Dex"
+description = "Instructions for installing Kubeflow with authentication support"
+weight = 4
++++
+
+This installation of Kubeflow is maintained by [Arrikto](https://www.arrikto.com/), is geared towards existing Kubernetes clusters and does not depend on any cloud-specific feature.
+
+In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and [Istio](https://istio.io/) for vendor-neutral authentication.
+
+![platform existing architecture](https://i.imgur.com/OlaN73j.png)
+
+### Prerequisites
+- Kubernetes Cluster with LoadBalancer support.
+
+If you don't have a Kubernetes Cluster, you can create a compliant Kubernetes Engine (GKE) on Google Cloud Platform cluster with the following script:
+
+<details>
+
+<summary>GKE Cluster Creation Script</summary>
+
+```bash
+#!/bin/bash
+
+set -e
+
+# This script uses the gcloud command.
+# For more info, visit: https://cloud.google.com/sdk/gcloud/reference/container/
+
+# Edit according to your preference
+GCP_USER="$(gcloud config list account --format "value(core.account)")"
+GCP_PROJECT="$(gcloud config list project --format "value(core.project)")"
+GCP_ZONE="us-west1-b"
+
+CLUSTER_VERSION="$(gcloud container get-server-config --format="value(validMasterVersions[0])")" 
+CLUSTER_NAME="kubeflow"
+
+############################
+# Create and setup cluster #
+############################
+
+gcloud container clusters create ${CLUSTER_NAME} \
+--project ${GCP_PROJECT} \
+--zone ${GCP_ZONE} \
+--cluster-version ${CLUSTER_VERSION} \
+--machine-type "n1-standard-8" --num-nodes "1" \
+--image-type "UBUNTU" \
+--disk-type "pd-ssd" --disk-size "50" \
+--no-enable-cloud-logging --no-enable-cloud-monitoring \
+--no-enable-ip-alias \
+--enable-autoupgrade --enable-autorepair
+
+echo "Getting credentials for newly created cluster..."
+gcloud container clusters get-credentials ${CLUSTER_NAME} --zone=${GCP_ZONE}
+
+echo "Setting up GKE RBAC..."
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=${GCP_USER}
+```
+
+</details>
+
+### Deploy Kubeflow
+
+If you are deploying Kubeflow on a multi-node cluster, you can follow this [guide](/docs/use-cases/kubeflow-on-multinode-cluster) to set up your system to use a remote NFS filesystem in cluster nodes.
+
+Follow these steps to deploy Kubeflow:
+
+1. Download a `kfctl` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/) and unpack it:
+
+    ```
+    tar -xvf kfctl_<release tag>_<platform>.tar.gz
+    ```
+
+    Alternatively, you can build the `kfctl` binary yourself:
```

Comment:
Should we drop the build kfctl yourself?
Building it yourself should be an advanced use case.
We should also have a single page with those instructions (or a markdown doc) so that we don't have multiple sets of instructions to update if it changes.

Reply:
done